src/modules/Message/utils/tokens/tokenize.ts

```diff
@@ -106,6 +179,7 @@ export function tokenizeMessage({
   mentionedUsers = [],
   templatePrefix = USER_MENTION_PREFIX,
 }: TokenParams): Token[] {
+  const { config } = useSendbirdStateContext();
```

Comment:
1. Do not use hooks outside of components or custom hooks.
2. Retrieve the value from `useSendbirdStateContext` where available, and then pass it as a parameter to the function.

Reply:
Yes, this is why unit tests were failing. Ill fix it as suggested!

Reply:
done!